### ISSUE TITLE
Interface defines in a Langium file isn't accessible from another

### ISSUE THREAD

**Issue by theogiraudet** [langium/langium#507]
When I try to separate my grammar and my model in two differents Langium File and add an import, I cannot return an element of one of my defined types. If I paste one of my interfaces in the grammar file, an error is thrown because the interface already exists on an imported file.

![image](https://user-images.githubusercontent.com/28688370/169250495-7e5d25f2-d0de-4aab-9efb-318bc2a5f645.png)
![image](https://user-images.githubusercontent.com/28688370/169250416-9de86b2d-5057-449d-8337-94baaee15c14.png)

I don't really know if this is a bug or an expected behaviour, but if it's intended, it's strange in this situation to throw error for an interface declared in another file.

Langium version: 0.3.0

## Steps To Reproduce

1. Create Hello World project
2. Replace the grammar with this (I keep default rules to avoid errors with Langium CLI):
```
grammar HelloWorld

import 'foo'

entry Model returns Foo:
    'model' name=ID;

Model2 infers Model:
    (persons+=Person | greetings+=Greeting)*;

Person:
    'person' name=ID;

Greeting:
    'Hello' person=[Person:ID] '!';

hidden terminal WS: /\s+/;
terminal ID: /[_a-zA-Z][\w_]*/;
terminal INT returns number: /[0-9]+/;
terminal STRING: /"[^"]*"|'[^']*'/;

hidden terminal ML_COMMENT: /\/\*[\s\S]*?\*\//;
hidden terminal SL_COMMENT: /\/\/[^\n\r]*/;
```
3. Create a new file `foo.langium` in the same directory than the existing grammar and paste this:
```
interface Foo {
    name: string
}
```

Link to code example:
https://github.com/theogiraudet/Minimal-Langium-Model-Error.git

## The current behavior
The interface is imported but cannot be used as return type.

## The expected behavior
The interface can either be used as a return type or be undetected, or display a custom message indicating that it is not possible.

**Comment by msujew** [comment]
@theogiraudet This is actually "intended" behavior in the sense that we deliberately released 0.3 without being able to cross reference types from other files, just to get the release out quickly. We plan to add this to 0.4 (likely next month).

**Comment by theogiraudet** [comment]
Okay, thanks for this information!
